tests/unit/handlers/test_structured_log.py

```diff
@@ -267,6 +270,41 @@ def test_format_with_request(self):
             for (key, value) in expected_payload.items():
                 self.assertEqual(value, result[key])
 
+    def test_format_with_traceparent(self):
+        import logging
+        import json
+
+        handler = self._make_one()
+        logname = "loggername"
+        message = "hello world，嗨 世界"
+        record = logging.LogRecord(logname, logging.INFO, "", 0, message, None, None)
+        expected_path = "http://testserver/123"
+        expected_agent = "Mozilla/5.0"
+        expected_trace = "4bf92f3577b34da6a3ce929d0e0e4736"
+        expected_span = "00f067aa0ba902b7"
+        trace_header = f"00-{expected_trace}-{expected_span}-09"
```

Comment:
nit: does flags=09 has a special meaning?

Reply:
Like above, I wanted to test with a flags field with multiple bits set (`b1001`), to make it more interesting. Most of the trace_sampled tests are in the `test__helpers.py` file.